### ISSUE TITLE
Safe Space and Doors creating lua errors

### ISSUE THREAD

**Issue by TXCReeseedrianed** [MattJeanes/Doors#29]
Whenever I tried spawning, it says the addon Doors is creating errors and the addon Safe Space is creating errors

**Comment by TheSenate720** [comment]
I've been having this issue as well. I don't have Safe Space but I do have Doors, and it's causing all sorts of errors. I think it really needs to be updated, as the last GMod or Tardis update might have broke it.

**Comment by AmyJeanes** [comment]
Can you share what errors you are getting exactly?

**Comment by TheSenate720** [comment]
> Can you share what errors you are getting exactly?

Yeah I can get you the script errors, gimme one sec. Sorry for the late reply, I have been very busy.

**Comment by TheSenate720** [comment]
[Doors] lua/entities/gmod_door_exterior/init.lua:59: bad argument #1 to 'SetModel' (string expected, got nil)
    1. SetModel - [C]:-1
        2. Initialize - lua/entities/gmod_door_exterior/init.lua:59
            3. unknown - lua/entities/gmod_tardis/init.lua:31
                4. Spawn - [C]:-1
                    5. SpawnFunction - lua/entities/gmod_door_exterior/init.lua:22
                        6. SpawnTARDIS - lua/tardis/metadata/sh_spawn_tardis.lua:60
                            7. unknown - lua/tardis/metadata/sh_spawn_tardis.lua:110
                                8. unknown - lua/includes/modules/concommand.lua:54
 
[Doors] lua/entities/gmod_door_exterior/init.lua:44: attempt to index field 'initqueue' (a nil value)
    1. func - lua/entities/gmod_door_exterior/init.lua:44
        2. unknown - lua/includes/extensions/net.lua:38

[Doors] lua/entities/gmod_door_exterior/init.lua:86: bad argument #1 to 'pairs' (table expected, got nil)
    1. pairs - [C]:-1
        2. unknown - lua/entities/gmod_door_exterior/init.lua:86

[Doors] lua/entities/gmod_door_exterior/modules/sh_interior.lua:109: bad argument #1 to 'pairs' (table expected, got nil)
    1. pairs - [C]:-1
        2. v - lua/entities/gmod_door_exterior/modules/sh_interior.lua:109
            3. CallHook - lua/entities/gmod_door_exterior/shared.lua:32
                4. CallHook - lua/entities/gmod_tardis/shared.lua:31
                    5. unknown - lua/entities/gmod_door_exterior/shared.lua:75

**Comment by TheSenate720** [comment]
These errors are causing the Tardis to not spawn.

**Comment by AmyJeanes** [comment]
Closing due to age, but let me know if you're still running into problems
